targets/kubernetes/scale-agent/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+images:
+  - name: armory/agent-k8s
+    newTag: 1.0.42 # Version of the agent to use. See https://docs.armory.io/docs/armory-agent/armory-agent-quick/#compatibility-matrix for available versions
+
+resources:
+  - scale-agent-service.yml
+  - scale-agent-deployment.yml
+
+patchesStrategicMerge:
+  - ./plugin-config.yml  
+
+secretGenerator:
+  - name: spin-secrets
+    options:
+      disableNameSuffixHash: true
+    literals:
+      - mysqlRootPassword=changeme
+      - mysqlCdPassword=changeme
+      - mysqlCdMigratePassword=changeme
+      - mysqlOrcaPassword=changeme
+      - mysqlOrcaMigratePassword=changeme
+      - mysqlF50Password=changeme
+      - mysqlF50MigratePassword=changeme
```

Comment:
```suggestion
```

We likely don't want to include these in the agent kustomization. I think when we move the sql config into the in-cluster component it will make more sense to define there. For now, this is fine 👍🏼